tests/core/test_infer.py

```diff
@@ -66,7 +68,9 @@ def test_infer_caching(self):
             size=1,
             dtype=torch.FloatTensor,
             batch_size=1,
-            output_ports={"dl_out": NeuralType({0: AxisType(BatchTag), 1: AxisType(BaseTag, dim=1)})},
+            output_ports={
+                "dl_out": NeuralType(ChannelType(), (AxisType(AxisKind.Batch), AxisType(AxisKind.Dimension, 1)))
```

Comment:
why is this not using axis=(...) notation?

Reply:
specifying kwarg's argument name is not required, here for brevity, I didn't use it